pkg/controller/import-controller.go

```diff
@@ -261,7 +267,16 @@ func (r *ImportReconciler) updatePvcFromPod(pvc *corev1.PersistentVolumeClaim, p
 				return err
 			}
 		}
+		anno[AnnBoundCondition] = "false"
```

Comment:
Isn't it kind of weird that we don't actually check that the PVC is bound?  Rather assume based on the fact that the pod is pending and scratch space is required?  Could be pending for other reasons.

Reply:
Good point, updated to check scratch PVC status, and put this condition in a different function.